src/app/beer_garden/api/http/authentication/__init__.py

```diff
@@ -49,9 +49,14 @@ def generate_access_token(user: User) -> str:
     jwt_headers = {"alg": "HS256", "typ": "JWT"}
     jwt_payload = {
         "sub": str(user.id),
-        "exp": datetime.utcnow() + timedelta(hours=12),
+        "exp": _get_token_expiration(),
```

Comment:
I changed this to a function call to make it easy to mock in my unit test.  It's otherwise identical to what was there.